exercises/palindrome/src/example.h

```diff
@@ -0,0 +1,19 @@
+#ifndef PALINDROME_H
+#define PALINDROME_H
+
+/* structure containing a pair of integer */
+typedef struct pair {
```

Comment:
AFAIK, you don't need `pair` in this line. Writing `typedef struct {` has the exact same effect.